services/ui-src/src/verbiage/pages/naaar/naaar-dashboard.ts

```diff
@@ -4,21 +4,36 @@ export default {
     body: [
       {
         type: "text",
-        content: "",
+        content:
+          "Complete one (1) report with information for applicable managed care plans and their applicable managed care programs. MMPs are considered both Medicaid and Medicare managed care plans and are not exempt from 42 CFR 438.207. Therefore, states must submit the tool for integrated plans; however, to reduce duplication, states can complete network adequacy sections of the tool (II.A.1-II.A.5) for Medicaid-only covered services. Reporting on Program of All-Inclusive Care for the Elderly (PACE) programs/plans is not required.",
```

Comment:
need link or link placeholder for `42 CFR 438.207` text as well

Reply:
✅ fixed!